<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText_0">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="configName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLabel_0">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Config name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandText_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLabel_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLabel_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLabel_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLabel_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
   <property name="tabChangesFocus" stdset="0">
    <bool>true</bool>
   </property>
   <property name="plainText" stdset="0">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>configName</tabstop>
  <tabstop>commandText_0</tabstop>
  <tabstop>comLabel_0</tabstop>
  <tabstop>commandText_1</tabstop>
  <tabstop>comLabel_1</tabstop>
  <tabstop>commandText_2</tabstop>
  <tabstop>comLabel_2</tabstop>
  <tabstop>commandText_3</tabstop>
  <tabstop>comLabel_3</tabstop>
  <tabstop>commandText_4</tabstop>
  <tabstop>comLabel_4</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
